website/content/docs/connect/ca/vault.mdx

```diff
@@ -210,30 +210,51 @@ The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
 Consul is granted read-only access to the PKI mount points and the Root CA, but is
 granted full control of the Intermediate or Leaf CA for Connect clients.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
+Please update those placeholders to match the path values in the CA provider configuration.
 
 <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
 ```hcl
-# Existing PKI Mounts
+# -----------------------------------------------------------------------------
+# Vault Managed PKI Mounts
+# -----------------------------------------------------------------------------
+
 path "/sys/mounts" {
```

Comment:
As of [this PR](https://github.com/hashicorp/consul/pull/12655/files#diff-7bf30611a760296e2e6ffcd7fe955b0eb8317fe0a43c470b8b52d311af4a2515L359), I don't think we actually need read capabilities on `sys/mounts` anymore... There's no other call to `Sys().ListMounts()` as far as I can tell. (We should actually test this first.)

Can we remove that from the docs? Or at least mark the version upon which it is no longer necessary?

Reply:
It looks like this change was made in Consul 1.12.0: https://github.com/hashicorp/consul/blob/main/CHANGELOG.md#1120-april-20-2022